core/storage/storage_check.go

```diff
@@ -9,24 +9,42 @@ import (
 	"github.com/rancher/agent/utilities/constants"
 	"github.com/rancher/agent/utilities/utils"
 	"golang.org/x/net/context"
+	"path/filepath"
+)
+
+const (
+	rancherSockDir = "/var/run/rancher/storage"
 )
 
 func IsVolumeActive(volume model.Volume, storagePool model.StoragePool, dockerClient *client.Client) (bool, error) {
 	if !isManagedVolume(volume) {
 		return true, nil
 	}
-	vol, err := dockerClient.VolumeInspect(context.Background(), volume.Name)
+	_, err := dockerClient.VolumeInspect(context.Background(), volume.Name)
 	if client.IsErrVolumeNotFound(err) {
 		return false, nil
 	} else if err != nil {
 		return false, errors.Wrap(err, constants.IsVolumeActiveError)
 	}
-	if vol.Mountpoint != "" {
-		return vol.Mountpoint != "moved", nil
-	}
 	return true, nil
 }
 
+func driverSockFile(volume model.Volume) string {
```

Comment:
Better specify it's for rancher-storage. e.g. `rancherStorageSockFile`.

Also, it's weird to generate this directory for non-rancher storage drivers. So instead of return `string`, we can return `string, error`, if it's not rancher-storage socket. So the function will generate and validate the socket location.